### PR TITLE
Feat/Release/2.3.0/SDK-1609

### PR DESCRIPTION
**Pull request by william-ct**
- Updated to [CleverTap Android SDK v4.6.6](https://github.com/CleverTap/clevertap-android-sdk/releases/tag/corev4.6.6)
- Updated to [CleverTap iOS SDK v4.1.6](https://github.com/CleverTap/clevertap-ios-sdk/releases/tag/4.1.6)
- Support for Exoplayer v2.17.1 - Only for Android.
- Adds new callback `CleverTapInboxItemClicked` to receive inbox item click.
- Adds a public method `deleteInboxMessagesForIds()` for deleting multiple App Inbox messages by passing a collection of messageIDs. Please note that this is only for iOS, and NO-OP for Android as of now.

### CODE REVIEW

**Comment by darshanclevertap** [comment]
@william-ct where are the changes for "Support for Exoplayer v2.17.1"? Which file?

**Review by darshanclevertap — APPROVED** [review]
@william-ct Approving this. But lets wait for iOS updates too

**Review by akashvercetti — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,18 @@
 Change Log
 ==========
 
+Version 2.3.0 *(xxx, 2023)*
```
Comment (darshanclevertap) [inline CHANGELOG.md:4]: @william-ct Please put appropriate date in the changelog for release

**Comment by akashvercetti** [comment]
@william-ct @kushagraa14 
- the PR should be merged to develop first.
- please generate the final updated .unitypackage file before merging.

**Review by darshanclevertap — APPROVED** [review]

**Review by darshanclevertap — APPROVED** [review]

File: src/Android/CleverTapUnityPlugin/clevertapsdk-unity/build.gradle [hunk 2]
```diff
@@ -51,7 +51,7 @@ dependencies {
     // This will add classes from com.unity3d.player to our library which will  allow the ClevertapPlugin.java class to get access to current unity activity and work on it accordingly
     compileOnly fileTree('libs')
 
-    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.6'
+    implementation 'com.clevertap.android:clevertap-android-sdk:4.6.7'
```
Comment (darshanclevertap) [inline src/Android/CleverTapUnityPlugin/clevertapsdk-unity/build.gradle:54]: @william-ct you will have to recreate the Unity package

**Review by darshanclevertap — APPROVED** [review]

**Review by akashvercetti — APPROVED** [review]

**Pull request merged**